.tx/config

```diff
@@ -1,14 +1,14 @@
 [main]
 host = https://www.transifex.com
 
-[edx-platform.edx-django-extensions]
-file_filter = edx-django-extensions/conf/locale/<lang>/LC_MESSAGES/django.po
-source_file = edx-django-extensions/conf/locale/en/LC_MESSAGES/django.po
+[edx-platform.edx-management-commands]
```

Comment:
There's a lot this repo doesn't need.

I actually wonder why we have this repo at all. What's the story here? Why weren't these commands integrated into edx-platform?